sulong/projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/LazyToTruffleConverterImpl.java

```diff
@@ -508,4 +508,8 @@ private void copyStructArgumentsToFrame(List<LLVMStatementNode> initializers, No
 
         return formalParamInits;
     }
+
+    public void resolveLinkageName() {
+        parser.parseLinkageName(runtime);
+    }
```

Comment:
I think this is not needed and makes it harder to understand. The caller (`LLVMParser`) can call `parser.parseLinkageName(runtime)` directly.